examples/orchestrators/adaptive_coding.py

```diff
@@ -47,6 +47,7 @@
         llm=llm,
         tools=[python_tool],
         role=AGENT_ROLE,
+        max_loops=4,
         inference_mode=InferenceMode.XML,
```

Comment:
With inference mode XML consistent problem with Parsing Action.
Works well with Default and Function calling.